bioconda_utils/docker_utils.py

```diff
@@ -469,3 +469,9 @@ def cleanup(self):
         if self.build_image and not self.keep_image:
             cmd = ['docker', 'rmi', self.docker_temp_image]
             utils.run(cmd, mask=False)
+
+
+def purgeImage(target, img):
+    image = "{}/{}".format(target, img)
```

Comment:
```suggestion
    image = "quay.io/{}/{}".format(target, img[::-1].replace("=", ":", 1)[::-1])
```
or
```suggestion
    image = "quay.io/{}/{}".format(target, ":".join(img.rsplit("=", 1)), )
```
I think. I'll check on the `quay.io` part.